src/locale/locales/fr/messages.po

```diff
@@ -711,21 +669,21 @@ msgstr "cliquez ici"
 #: src/components/RichText.tsx:189
 #: src/components/TagMenu/index.web.tsx:125
 msgid "Click here to open tag menu for {tag}"
-msgstr ""
+msgstr "Cliquez ici pour ouvrir le menu d'étiquette pour {tag}"
```

Comment:
Little nitpick: we used `’` all around the translation instead of `'`, as it should be according to typographical best practices in French. (but it's still a nitpick, honestly, nobody notices this…)

```suggestion
msgstr "Cliquez ici pour ouvrir le menu d’étiquette pour {tag}"
```